src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, num) {
+      this.result = callback(this.result, num);
+
+      return this;
+    },
+
+    add(a, b) {
```

Comment:
The 'add' function should only take one argument, not two. It should add the argument to the current value of 'result', not add two numbers together.